frontend/public/models/index.ts

```diff
@@ -696,8 +696,8 @@ export const ClusterServiceBrokerModel: K8sKind = {
 };
 
 export const ClusterServiceClassModel: K8sKind = {
-  label: 'Cluster Service Class',
-  labelPlural: 'Cluster Service Classes',
+  label: 'Service Class',
+  labelPlural: 'Service Classes',
```

Comment:
Need to leave "Cluster" in these names. There are also regular "Service Class" resources that are namespace-scoped, which we'll need to support.

Reply:
Note that the model should still have "Cluster" in the label. Only the nav label should be different (since the page will show both cluster service classes and service classes).